### PR TITLE
Add Meshery Operator project logo and description

### PR DESCRIPTION
**Pull request by Anita-ihuman**
fix for this issue "https://github.com/layer5io/layer5/issues/523":
which was updated to include the meshery operator project  logo and  its description
signed-off-by: Anita Ihuman <charlesanita403@gmail.com>

**Description**

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Anita-ihuman** [comment]
please do well to review this request and merge it. Thank you

**Comment by Anita-ihuman** [comment]
Good evening can you please review my pull request @leecalcote🥺🙏🏼

**Comment by Nikhil-Ladha** [comment]
Hey @Anita-ihuman I think you have missed to sign off your PR, if you can make an update to that. We will be good to move ahead.

**Comment by Nikhil-Ladha** [comment]
Also, if you can just follow the format for other projects mentioned below in the readme and add this project in the same way at last instead of adding it at the beginning.

**Comment by Anita-ihuman** [comment]
please review changes made Thank you.

File: README.md [hunk 1]
```diff
@@ -21,16 +21,6 @@ We create and steward initiatives that press on the service mesh-centric envelop
...
-<a href="https://meshery.io/community"><img alt="Layer5 Service Mesh Community" src="img/readme/meshery-operator.svg" style="margin:10px;" width="110px" align="left" /></a>
-# Meshery Operator
-Meshery Operator is the multi-service mesh operator and implementation of MeshSync.
-<br />
-<br />
-<br />
-</p>
-
-
 <p style="clear:both;">
```
Comment (Nikhil-Ladha) [inline README.md]: You can follow this way of adding the project description for Meshery Operator as well, to maintain consistency.

**Comment by Nikhil-Ladha** [comment]
Hey @Anita-ihuman , the DCO check is still failing. If you take a look into that again.
Also, if you add the operator in the same way as we have other projects listed it will maintain the consistency in the file.

<p style="clear:both;">
<h2><a href="https://github.com/layer5io/meshery-operator">Meshery Operator</a></h2>
<a href="https://meshery.io/community">Meshery Operator</a>
<a href="https://github.com/meshery-operator">Meshery Operator</a> is the multi-service mesh operator and implementation of MeshSync.
<br /><br /><br />
</p>


P.S: Since we don't have meshey-operator listed in the site yet, you can add the link to GitHub repo of it, for the time being, we can update later.
// @leecalcote

**Comment by PluckyPrecious** [comment]
Hi @Anita-ihuman you can use this easy method to sign DCO; ` git commit --amend --signoff`  and then `git push --force-with-lease origin feature/username/issue`. The feature is your "branch", "your-username" and "issue number", hope it helps. 😅

**Pull request closed without merging**